pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -1101,6 +1126,65 @@ pub mod pallet {
 				},
 			}
 		}
+
+		#[pallet::call_index(22)]
+		#[pallet::weight({1000})]
+		pub fn add_affiliation(origin: OriginFor<T>, affiliate_id: u64) -> DispatchResult {
```

Comment:
u64 is really big, couldn't we use the u32 ... it's more then enough. It will make the reffereal code easier